docs/guides/hooks.md

```diff
@@ -0,0 +1,85 @@
+# Hooks
+Hooks exist so that users can latch on to certain Video.js lifecycle events
```

Comment:
I think we ought to explain that this is a separate concept from per-player events. In fact, we should probably not use the term "events" at all, lest we cause confusion with events-events.